_commands/basics/kill.md

```diff
@@ -0,0 +1,57 @@
+---
+---
+
+kill
+--
+
+`kill` is used to send signal to a process or to kill a process. 
+We typically use kill -SIGNAL PID, where you know the PID of the process.
+ 
+
+~~~ bash
+$ kill -9 7243
```

Comment:
If you just ``kill PID`` it sends SIGTERM, which gives the process a chance to clean up after itself.

Ref: http://www.gnu.org/software/libc/manual/html_node/Termination-Signals.html